.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Enhancement
+about: Suggest an idea for this project
+title: "  [ENHANCEMENT]<Enter comprehensive title>"
+labels: enhancement
+assignees: ''
+
```

Comment:
printer type: {MK3S, MK3, MK2.5S, MK2.5, MK2S, MK2}
upgrade: {MMU2S, MMU2, MMU1}

---
As a summary, please make sure you have ticked all points on this
checklist:
-->
* [ ] I searched in open and closed issues and couldn't find anything that matches.
* [ ] Printer type (MK3S, MK3, MK2.5S, MK2.5, MK2S, MK2)
* [ ] Upgrade (MMU2S, MMU2, MMU1)